exercises/leap/src/main/scala/Leap.scala

```diff
@@ -0,0 +1,3 @@
+object Year {
```

Comment:
Now this touches an interesting subject:
Some time ago we decided in issue #137 to have such a file with the expected functions for the first 10 exercises.

`leap` was originally not among those, but now after the reordering according to difficulty it is.
So I think to add this file is just fine. Am I right, @ricemery @ErikSchierboom ? Maybe we should also review the other affected exercises under this aspect?